src/main/java/net/fabricmc/meta/web/ServerBootstrap.java

```diff
@@ -75,6 +76,10 @@ private static Handler boostrapHandler() {
         String version = ctx.pathParam(name);
 
         for (V v : versions) {
+            if (version.equals("stable") && v.isStable()) {
```

Comment:
I think we said this shouldnt work for the minecraft version.

Reply:
I brought this up on Discord but didn't tag you, sorry.

That's easily doable here by adding ` && !name.equals("game_version")` to the conditional, but why not leave the code simpler and let the admin dig their own grave if they want to auto-update the Minecraft version?

Reply:
Fair enough. 👍